binderhub/templates/index.html

```diff
@@ -56,6 +56,7 @@ <h4 id="form-header" class='row'>Build and launch a repository</h4>
                 <li class="dropdown-item" value="git"><a href="#">Git repository</a></li>
                 <li class="dropdown-item" value="zenodo"><a href="#">Zenodo DOI</a></li>
                 <li class="dropdown-item" value="figshare"><a href="#">Figshare DOI</a></li>
+                <li class="dropdown-item" value="hydroshare"><a href="#">Hydroshare resource</a></li>
```

Comment:
I am unsure about the wording here, because for Zenodo and Fishare, we also support plain Zenodo URLs (they are just patched through), and it seems the Hydroshare support in repo2docker also supports DOIs...

@betatim What wording should BinderHub use here? Favour "DOI" or "resource" (or "record"). I'd tend towards only talking about DOIs externally, but not failing when direct resource URLs are provided.

Reply:
I used resource because it is a HydroShare term.  I'm fine highlighting the DOI, but I do think it'd be good to point to a resource id as well, because resources don't have a functional DOI until they publish the resource.  Let me know what you'd like, and I can update it in this PR

Reply:
I think if there's a provider-specific wording, we can just use that since a user familiar with it will be expecting that word (as long as it's not too long or complex)